lookout/style/format/analyzer.py

```diff
@@ -136,8 +77,8 @@ def __init__(self, model: FormatModel, url: str, config: Mapping[str, Any]) -> N
         """
         super().__init__(model, url, config)
         self._log = logging.getLogger(type(self).__name__)
-        self.config = self._load_analyze_config(self.config)
-        with open(self.config["comment_template"], encoding="utf-8") as f:
+        self.config = self._load_config(config)
+        with open(self.config["common"]["comment_template"], encoding="utf-8") as f:
```

Comment:
Why do we need the comment template in "common"?

Reply:
"common" is also common for all languages. If I place `comment_template` to `analyze` I should get it for each language specifically:
```
with open(self.config["common"]["javascript"]["comment_template"]
```
 That means I should move comment template loading from `__init__` to internals. 
I can do it if you think it is better.

Reply:
Not really, "common" is something which is shared between "analyze" and "train". Otherwise, we have "global" inside "analyze" and "train" - btw, we should probably rename it to "common" for clarity.

Reply:
In this case I expect the setting to go into "analyze/global"

Reply:
"analyze/global" is global for all languages, yes, but not "common". "analyze/global" is removed to have a structure like: 
```
"analyze": {
    "<lang1>": {},
    "<lang2>": {},
    "<lang3>": {},
}
```
We cannot just keep it because If someone decide to override `comment_template` for one paticular language (and it is possible if we place in "analyze/global") we are able to load only one global template. 

So "global" and "common" are different and I'd like to not have "analyze/common" and "train/common" because it is just make things more complex.

Reply:
Just to clarify "analyze/global" contains default values for all languages. When we replace unspecified values with defaults we remove "analyze/global"

Reply:
The high level structure must be this: "analyze"- all configuration for `analyze()`. "train" - all configuration for `train()`. "common" must be used for configuration which is shared between train and analyzer, and has nothing to do with languages.

The comment template is not used in `train()` and cannot be put inside "common".